src/util/range/bitmap_range.c

```
@@ -77,25 +72,26 @@ bool BitmapRange_FromRanges (
 	ASSERT(range  != NULL);
 	ASSERT(ranges != NULL);
 
-	// start with [0..node_count) range
-    // size_t node_count = Graph_UncompactedNodeCount(QueryCtx_GetGraph());
-    roaring64_bitmap_add_range_closed(range, min, max);
+	// clear range
+	roaring64_bitmap_remove_range_closed(range, min, max);
```

Cppcheck (2.19.0)

[error] Memory leak

(memleak)